inc/site-editor.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Customizations for the site editor.
+ *
+ * @package Create_WordPress_Theme
+ */
+
+namespace Create_WordPress_Theme\Site_Editor;
+
+use Alley\WP\Caper;
+
+add_action( 'after_setup_theme', __NAMESPACE__ . '\action__after_setup_theme' );
+
+/**
+ * Prevents access to the Site Editor on environments other than local and develop.
+ */
+function action__after_setup_theme(): void {
+	global $pagenow;
+
+	if ( ! in_array( wp_get_environment_type(), [ 'local', 'development' ], true ) ) {
+		// Caper is being installed via Create WordPress Project. If it's not available, bail.
+		if ( ! class_exists( 'Alley\WP\Caper' ) ) {
+			return;
+		}
+
+		Caper::deny_to_all()->primitives( 'edit_theme_options' );
+
+		if ( 'site-editor.php' !== $pagenow ) {
```

Comment:
If we are blocking access to `site-editor.php` entirely then this condition will always be true.

I propose removing Caper from this entirely, and just removing the menu item and redirecting if someone tries to hit it directly. If we find that we need to introduce additional capabilities checks in the future, we can always add them back in.